### ISSUE TITLE
FAQ link leads nowhere

### ISSUE THREAD

**Issue by saiichihashimoto** [greenkeeperio/website#28]
When I'm on the faq: https://greenkeeper.io/faq.html

And I click this link:
![image](https://cloud.githubusercontent.com/assets/2819256/18610624/b24cf7e2-7cd5-11e6-8927-0769ec90198a.png)

It takes me here:
https://greenkeeper.io/faq.html#github-activity

And there's no section on the page that leads to answering "Greenkeeper is creating branches and immediately deleting them again".

**Comment by ntwb** [comment]
Yes there is, it clearly states:

> "This might be because the dependency updates are within your specified ranges, and not breaking your tests. So there’s no point in bothering you with a PR, since everything still works."
>
> "You’ll only receive pull requests when an in-range dependency breaks your build, or when a dependency updates outside your specified range."

Hence it creates a branch, runs the tests, if covered by your semver ranges it deletes the branch without creating a PR

**Comment by saiichihashimoto** [comment]
OH, now I get it. What greenkeeper does is _almost_ what I thought it did (and is actually much cooler). Neat. :-)

The reason why I created two issues is because:

1. The other issue, where I thought greenkeeper was broken. It's not and I'm wrong.
2. This issue, where it's super confusing that clicking `Greenkeeper is creating branches and immediately deleting them again` leads to `I ENABLED A REPO, BUT NOTHING’S HAPPENING / DELETED BRANCHES` since it seems like this was meant for the link right before, `I enabled a repo, but nothing’s happening`.

Clicking a link and having it direct to me a section with a totally different title when there's a link that's supposed to lead there is very confusing. I just assumed a section was removed instead of reading the text to realize it wasn't a mistake. Maybe I'm just lazy, but it just seems like a section is missing.
